otherlibs/stdune/src/copyfile_stubs.c

```diff
@@ -90,7 +90,7 @@ CAMLprim value stdune_sendfile(value v_in, value v_out, value v_size) {
   caml_release_runtime_system();
   /* TODO Use copy_file_range once we have a good mechanism to test for its
    * existence */
-  int ret = dune_sendfile(FD_val(v_in), FD_val(v_out), Int_val(v_size));
+  int ret = dune_sendfile(FD_val(v_in), FD_val(v_out), Long_val(v_size));
```

Comment:
FTR, I checked whether it's safe to use `size_t` with `Long_val`; the ocaml runtime does that so it seems good.